tests/performance/results.js

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const testSuites = [
+    'front-end-classic-theme',
+    'front-end-block-theme',
```

Comment:
Maybe change the file names to a more specific `home-page-*` instead of generic `front-end-*`? Or maybe even more precise `home-*` to indicate that this is the blog (`is_home()`) template being tested (to separate from a potential `front-page-*` in the future)?

Reply:
Thanks Felix for the feedback. I have updated name as suggested.